EU/main.yaml

```
@@ -81,3 +102,17 @@ sources:
       simulation_id: d3hp003
       time_start: 2020-01-01T00:00:00
       time_end: 2021-03-01T00:00:00
+      title: ICON 2.5 km simulation
+      summary: |
+        Atmosphere-only simulation at 2.5 km resolution. Run for 1 year and 2 months starting in 2020. 
```

To render "Processing" in a new paragraph, an empty line is needed. 
```suggestion
        Atmosphere-only simulation at 2.5 km resolution. Run for 1 year and 2 months starting in 2020. 

```